models/registration_management.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (sequelize, Sequelize) => {
+  const Registration_management = sequelize.define('registration_management', {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    project_registration_open: {
+      type: Sequelize.BOOLEAN
+    },
+    project_registration_message: {
+      type: Sequelize.STRING
+    },
+    topic_registration_open: {
+      type: Sequelize.BOOLEAN
+    },
+    topic_registration_message: {
+      type: Sequelize.STRING
+    }
+  })
```

Comment:
Just a heads up, by default, Sequelize uses `camelCase` for column names. The timestamp columns (`createdAt` and `updatedAt`) will be in camelCase here. Let's address that issue in bulk at a later time.